### PR TITLE
[Merged by Bors] - chore: deal with porting notes about 'protected'

### PR DESCRIPTION
**Pull request by kim-em**

### CODE REVIEW

File: Mathlib/Algebra/GroupWithZero/Divisibility.lean [hunk 1]
```diff
@@ -139,9 +139,6 @@ theorem dvd_antisymm : a ∣ b → b ∣ a → a = b := by
   rw [mul_assoc, eq_comm, mul_right_eq_self₀, mul_eq_one] at hcd
   obtain ⟨rfl, -⟩ | rfl := hcd <;> simp
 
--- Porting note: `attribute [protected]` is currently unsupported
--- attribute [protected] Nat.dvd_antisymm --This lemma is in core, so we protect it here
-
```
Comment (jcommelin) [inline Mathlib/Algebra/GroupWithZero/Divisibility.lean:144]: Why do we not want to protect it anymore? Should the lemma below be protected? Or should it at least get a comment?

File: Mathlib/NumberTheory/FunctionField.lean [hunk 2]
```diff
@@ -54,7 +54,6 @@ Note that `F` can be a function field over multiple, non-isomorphic, `Fq`.
 abbrev FunctionField [Algebra (RatFunc Fq) F] : Prop :=
   FiniteDimensional (RatFunc Fq) F
 
--- Porting note: Removed `protected`
```
Comment (jcommelin) [inline Mathlib/NumberTheory/FunctionField.lean:57]: Just flagging: we're deciding to not add it back. Which is fine with me.

File: Mathlib/Order/Filter/Germ/Basic.lean [hunk 3]
```diff
@@ -79,7 +79,6 @@ def productSetoid (l : Filter α) (ε : α → Type*) : Setoid ((a : _) → ε a
 
 /-- The filter product `(a : α) → ε a` at a filter `l`. This is a dependent version of
   `Filter.Germ`. -/
--- Porting note: removed @[protected]
```
Comment (jcommelin) [inline Mathlib/Order/Filter/Germ/Basic.lean:82]: Just flagging: we're deciding to not add it back. Which is fine with me.

File: Mathlib/Topology/Algebra/Group/Basic.lean [hunk 4]
```diff
@@ -589,7 +589,6 @@ protected theorem Inducing.topologicalGroup {F : Type*} [Group H] [TopologicalSp
     toContinuousInv := hf.continuousInv (map_inv f) }
 
 @[to_additive]
--- Porting note: removed `protected` (needs to be in namespace)
```
Comment (jcommelin) [inline Mathlib/Topology/Algebra/Group/Basic.lean:592]: Just flagging: we're deciding to not add it back. Which is fine with me.

File: Mathlib/Topology/MetricSpace/Dilation.lean [hunk 5]
```diff
@@ -267,7 +267,7 @@ protected def id (α) [PseudoEMetricSpace α] : α →ᵈ α where
 instance : Inhabited (α →ᵈ α) :=
   ⟨Dilation.id α⟩
 
-@[simp] -- Porting note: Removed `@[protected]`
+@[simp]
 theorem coe_id : ⇑(Dilation.id α) = id :=
```
Comment (jcommelin) [inline Mathlib/Topology/MetricSpace/Dilation.lean:271]:
```suggestion
@[simp]
protected theorem coe_id : ⇑(Dilation.id α) = id :=
```
This seems like a reasonable one to protect, I think.

File: Mathlib/Algebra/GroupWithZero/Divisibility.lean (hunk 1 above)
Comment (kim-em) [reply Mathlib/Algebra/GroupWithZero/Divisibility.lean:144]: It is protected in Lean now.

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**